postgres/README.md

```diff
@@ -9,70 +9,122 @@ Get metrics from postgres service in real time to:
-* **`tags`** (Optional) - A list of tags applied to all metrics collected. Tags may be simple strings or key-value pairs.
-* **`relations`** (Optional) - By default, all schemas are included. Add specific schemas here to collect metrics for schema relations. Each relation will generate 10 metrics and an additional 10 metrics per index. Use the following structure to declare relations:
-
-```
-relations:
-  - relation_name: my_relation
-    schemas:
-      - my_schema_1
-      - my_schema_2
-```
-
-* **`collect_function_metrics`** (Optional) - Collect metrics regarding PL/pgSQL functions from pg_stat_user_functions
-* **`collect_count_metrics`** (Optional) - Collect count metrics. The default value is `True` for backward compatibility, but this might be slow. The recommended value is `False`.
+#### Metric Collection
+
+1. Edit the `postgres.yaml` file to point to your server and port, set the masters to monitor. See the [sample postgres.yaml](https://github.com/DataDog/integrations-core/blob/master/postgres/conf.yaml.example) for all available configuration options. Configuration Options:
+  
+  * **`username`** (Optional) - The user account used to collect metrics, set in the [Installation section above](#installation)
+  * **`password`** (Optional) - The password for the user account.
+  * **`dbname`** (Optional) - The name of the database you want to monitor.
+  * **`ssl`** (Optional) - Defaults to False. Indicates whether to use an SSL connection.
+  * **`use_psycopg2`** (Optional) - Defaults to False. Setting this option to `True` will force the Datadog Agent to collect PostgreSQL metrics using psycopg2 instead of pg8000. Note that pyscopg2 does not support SSL connections.
+  * **`tags`** (Optional) - A list of tags applied to all metrics collected. Tags may be simple strings or key-value pairs.
+  * **`relations`** (Optional) - By default, all schemas are included. Add specific schemas here to collect metrics for schema relations. Each relation will generate 10 metrics and an additional 10 metrics per index. Use the following structure to declare relations:
+
+  ```
+  relations:
+    - relation_name: my_relation
+      schemas:
+        - my_schema_1
+        - my_schema_2
+  ```
+
+  * **`collect_function_metrics`** (Optional) - Collect metrics regarding PL/pgSQL functions from pg_stat_user_functions
+  * **`collect_count_metrics`** (Optional) - Collect count metrics. The default value is `True` for backward compatibility, but this might be slow. The recommended value is `False`.
+
+2. [Restart the Agent](https://docs.datadoghq.com/agent/faq/start-stop-restart-the-datadog-agent) to start sending Postgres metrics to Datadog.
+
+#### Log Collection
+
+Postgres default logging is to stderr and logs do not contains much information. This is why we suggest to log into a file with a specific log prefix. 
```

Comment:
logs do not contains much information. -> logs do not include detailed information.
with a specific log prefix -> with additional details specified in the log line prefix

Reply:
👍